src/mixins/onThisPageRegistrator.js

```diff
@@ -57,6 +57,11 @@ export default {
       if (primaryContentSections) {
         primaryContentSections.forEach((section) => {
           switch (section.kind) {
+          case SectionKind.mentions:
+            this.store.addOnThisPageSection({
+              title: 'Mentioned in', anchor: anchorize('mentions'), level: 2,
```

Comment:
We should use the localization helper here as well once you add an entry for this string.